src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormHelperText,
+  IconButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  TableCell,
+  TableRow,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { DatePicker } from '@mui/x-date-pickers';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import prettyBytes from 'pretty-bytes';
+import { Field, FieldProps, ErrorMessage, useFormikContext } from 'formik';
+import { AssociatedFile, LicenseNames, licenses } from '../../../types/associatedArtifact.types';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { SpecificFileFieldNames } from '../../../types/publicationFieldNames';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { TruncatableTypography } from '../../../components/TruncatableTypography';
+
+interface FilesTableRowProps {
+  file: AssociatedFile;
+  removeFile: () => void;
+  toggleLicenseModal: () => void;
+  baseFieldName: string;
+}
+
+export const FilesTableRow = ({ file, removeFile, baseFieldName }: FilesTableRowProps) => {
+  const { t } = useTranslation();
+  const [openConfirmDialog, setOpenConfirmDialog] = useState(false);
+  const toggleOpenConfirmDialog = () => setOpenConfirmDialog(!openConfirmDialog);
+  const { setFieldValue, setFieldTouched } = useFormikContext();
+
+  return (
+    <TableRow data-testid={dataTestId.registrationWizard.files.fileRow}>
+      <TableCell sx={{ wordBreak: 'break-word' }}>
```

Comment:
Skulle vi satt en minWidth også for name og embargo?
Slik at vi vet at embargo date får plass, og at vi ikke bryter opp filnavn som har få tegn
![bilde](https://user-images.githubusercontent.com/6313468/203984559-7b98f28f-918a-4b28-9346-ba4e6f70926a.png)